apps/studio/components/layouts/ProjectLayout/LayoutHeader/LocalVersionPopover.tsx

```diff
@@ -0,0 +1,168 @@
+import dayjs from 'dayjs'
+
+import { DocsButton } from 'components/ui/DocsButton'
+import { InlineLink } from 'components/ui/InlineLink'
+import { useDockerHubStudioVersionsQuery } from 'data/misc/docker-hub-versions-query'
+import { STUDIO_VERSION } from 'lib/constants'
+import {
+  Badge,
+  Button,
+  Dialog,
+  DialogContent,
+  DialogHeader,
+  DialogSection,
+  DialogTitle,
+  DialogTrigger,
+  Popover_Shadcn_,
+  PopoverContent_Shadcn_,
+  PopoverSeparator_Shadcn_,
+  PopoverTrigger_Shadcn_,
+  SimpleCodeBlock,
+  Tabs_Shadcn_,
+  TabsContent_Shadcn_,
+  TabsList_Shadcn_,
+  TabsTrigger_Shadcn_,
+} from 'ui'
+import { Admonition } from 'ui-patterns'
+
+export const LocalVersionPopover = () => {
+  const { data } = useDockerHubStudioVersionsQuery()
+  const hasLatestVersionCheck = !!data?.latest
+  const isLatestVersion = data?.latest === STUDIO_VERSION
+
+  const approximateNextRelease = !!data?.last_updated_at
+    ? dayjs(data?.last_updated_at).utc().add(14, 'day').format('DD MMM YYYY')
+    : undefined
+
+  return (
+    <Popover_Shadcn_ modal={false}>
+      <PopoverTrigger_Shadcn_ className="flex items-center">
+        <Badge variant={isLatestVersion ? 'default' : 'brand'}>
+          {isLatestVersion ? 'Latest' : 'Update available'}
+        </Badge>
+      </PopoverTrigger_Shadcn_>
+      <PopoverContent_Shadcn_ align="end" className="w-80 px-0">
+        {hasLatestVersionCheck ? (
+          !isLatestVersion ? (
+            <div className="px-4 mb-3">
+              <p className="text-sm mb-2">
```

Comment:
Might need color class

Reply:
intention is for this to just be text-foreground though